### ISSUE TITLE
EER question

### ISSUE THREAD

**Issue by hash2430** [cvqluu/Additive-Margin-Softmax-Loss-Pytorch#2]
Hello. I am Sunghee Jung.
I guess this repository is about image verification..
Any chance you are working on speaker verification version of this repo?
